src/core/Song.cpp

```diff
@@ -1231,16 +1222,6 @@ void Song::exportProjectTracks()
 
 void Song::exportProject( bool multiExport )
 {
-	if( isEmpty() )
-	{
-		QMessageBox::information( gui->mainWindow(),
-				tr( "Empty project" ),
```

Comment:
Am I reading this correctly, these project empty warnings are no longer issued, and we now export and empty song?
If so has this been tested?